README.md

```diff
@@ -403,6 +403,31 @@ Destination[] directDestinations = amadeus.airport.directDestinations.get(Params
 // body can be a String version of your JSON or a JsonObject
 FlightAvailability[] flightAvailabilities
   = amadeus.shopping.availability.flightAvailabilities.post(body);
+
```

Comment:
I don't think that's the right place to add the example. Maybe in another section of the README such as the "Response", we could mention the support of dictionaries and meta: 
<img width="876" alt="Screenshot 2022-02-23 at 13 10 41" src="https://user-images.githubusercontent.com/15012967/155317006-628d9ee2-f4a3-49a3-9465-cb9a60f5248b.png">